provisions/roles/db/tasks/main.yml

```diff
@@ -1,12 +1,13 @@
 ---
 - name: Ensure cccp data dir exists
-  file: path=/var/lib/cccp/db state=directory
+  file: path= "{{ db_backup_host_path }}" state=directory
   sudo: yes
   tags: db
+  when: not test
```

Comment:
Does this mean that when we're doing a CI deployment, the backup directory won't be created? But it'll still be created in DevCloud, right? How about making it something like `when: production` so that it's created only for prod? We don't need it for either DevCloud or CI.

Reply:
@dharmit : But for testing purpose we need to make sure the database also works, the database is going to have mutiple functionality on top of it, like APIs UI etc, the host directory availability for the service should be less/no-overhead to developer, it should be taken care by provisioning itself.